params/config.go

```diff
@@ -34,7 +34,7 @@ var (
 
 	BSCGenesisHash    = common.HexToHash("0x0d21840abff46b96c84b2ac9e10e4f5cdaeb5693cb665db62a2f3b02d2d57b5b")
 	ChapelGenesisHash = common.HexToHash("0x6d3c66c5357ec91d5c43af47e234a939b22557cbb552dc45bebbceeed90fbe34")
-	RialtoGenesisHash = common.HexToHash("0xaabe549bfa85c84f7aee9da7010b97453ad686f2c2d8ce00503d1a00c72cad54")
+	RialtoGenesisHash = common.HexToHash("0x6b5cf721c57dec67700d664c98c4f779b7bb980301b9a649842ba584fd72dd6c")
```

Comment:
why change this?

Reply:
Edit this for the QA test, after resetting the QA env, this genesis hash would change.